src/components/list/demoListControls/index.html

```diff
@@ -0,0 +1,29 @@
+<md-list ng-controller="ListCtrl">
+  <md-subheader class="md-no-sticky">Single Action Checkboxes</md-subheader>
+  <md-item ng-repeat="topping in toppings">
+    <p> {{ topping.name }} </p>
+    <md-checkbox class="md-secondary" ng-model="topping.wanted" aria-label="{{ topping.name }}">
+  </md-item>
+  <md-subheader class="md-no-sticky">Clickable Items with Secondary Controls</md-subheader>
+  <md-item ng-click="navigateTo(setting.extraScreen)" ng-repeat="setting in settings">
+    <md-icon md-svg-icon="{{setting.icon}}"></md-icon>
+    <p> {{ setting.name }} </p>
+    <md-switch class="md-secondary" ng-model="setting.enabled" aria-label="{{ setting.name }}">
+  </md-item>
+  <md-item ng-click="navigateTo('data usage')">
+    <md-icon md-svg-icon="cached"></md-icon>
+    <p>Data Usage</p>
+  </md-item>
+  <md-subheader class="md-no-sticky">Checkbox with Secondary Action</md-subheader>
+  <md-item ng-repeat="message in messages">
+    <md-checkbox ng-model="message.selected" aria-label="Select"></md-checkbox>
```

Comment:
When I click on an `md-checkbox` to activate it, it gets different indentation. Seems like a weird render bug, because it goes away when I blur the browser window. 
![Checkboxes](https://cloud.githubusercontent.com/assets/1045233/6813553/63d1f6a6-d233-11e4-9592-0bd8630748f2.png)